applications/asset_tracker_v2/src/carrier_certs/carrier_certs.c

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Looks like this file contains boilerplate code used when integration carrier_lib. Maybe a future expansion would be to move this to the OS integration layer?